app/src/main/res/layout/lessons_in_progress_chapter_view.xml

```diff
@@ -34,7 +34,7 @@
         android:layout_gravity="center"
         android:layout_marginStart="10dp"
         android:fontFamily="sans-serif"
-        android:importantForAccessibility="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? View.IMPORTANT_FOR_ACCESSIBILITY_YES : View.IMPORTANT_FOR_ACCESSIBILITY_NO}"
+        android:importantForAccessibility="yes"
```

Comment:
Do we need to have ``importantForAccessibility`` if it's a text view? I thought that they are always important by default.

Reply:
Yes I think we can remove this also.
What @rt4914 you suggest in this case?

Reply:
Ahh, completely missed on this thought process. Earlier we were using conditions because it was one single file but now that we have multiple files we can actually remove this condition from all those files where importantForAccessibility=yes and can keep this wherever importantForAccessibility=no.